### PR TITLE
Adding small fixes

### PR DESCRIPTION
**Pull request by PythooonUser**
## Summary
Adding small fixes, like:
- Beautifying various comments
- Name change of _outbound_send_rate to make it match with _inbound_polling_rate (or is this incorrect in English?)
- Updating README.md with a deinstallation section and make introductions
- Converting rates to proper times
- Making method send_events a private one because we now have wrappers for that and do not support other commands yet. So there is not need by the user to call it directly
- Increasing twitch-observer version to 0.1.0 in `__init__.py`

### CODE REVIEW

**Comment by PythooonUser** [comment]
Maybe I goofed something up, but most of those commits have been merged to your branch already, but GitHub thinks they're not.

File: twitchobserver/twitchobserver.py [hunk 1]
```diff
@@ -92,8 +92,9 @@ def get_events(self):
 
         return result
 
-    def send_events(self, *events):
-        """Queues up the given events for sending"""
+    def _send_events(self, *events):
```
Comment (JoshuaSkelly) [inline twitchobserver/twitchobserver.py]: 👍 Good call

File: twitchobserver/twitchobserver.py [hunk 2]
```diff
@@ -209,7 +208,7 @@ def outbound_worker():
 
             while self._is_running:
                 try:
-                    if self._outbound_event_queue and time.time() - self._last_time_sent > self._outbound_send_rate:
+                    if self._outbound_event_queue and time.time() - self._last_time_sent > (1 / self._outbound_sending_rate):
```
Comment (JoshuaSkelly) [inline twitchobserver/twitchobserver.py]: Why are you inverting this?

File: twitchobserver/twitchobserver.py [hunk 3]
```diff
@@ -218,7 +217,7 @@ def outbound_worker():
 
                         self._last_time_sent = time.time()
 
-                    time.sleep(self._outbound_send_rate / 2)
+                    time.sleep(2 / self._outbound_sending_rate)
```
Comment (JoshuaSkelly) [inline twitchobserver/twitchobserver.py]: This is incorrect.

File: twitchobserver/twitchobserver.py [hunk 4]
```diff
@@ -242,7 +241,7 @@ def stop(self, force_stop=False):
 
         # Wait until all outbound events are sent
         while self._outbound_event_queue and not force_stop:
-            time.sleep(self._outbound_send_rate)
+            time.sleep(1 / self._outbound_sending_rate)
```
Comment (JoshuaSkelly) [inline twitchobserver/twitchobserver.py]: Again, why are you inverting the rates here? The calculation of the poll rate should be correct.

File: README.md [hunk 5]
```diff
@@ -94,6 +101,21 @@ observer.join_channel('channel')
 observer.leave_channel('channel')
 ```
 
+### 5. Deinstallation
```
Comment (JoshuaSkelly) [inline README.md]: Do we need this?

File: README.md [hunk 6]
```diff
@@ -9,19 +9,26 @@ Turn Twitch chatter into Python events.
...
+To install twitch-observer clone from the [repo](https://github.com/JoshuaSkelly/twitch-observer) and run:
 
 ```
 $ pip install .
 ```
+
+You can also use the makefile by typing:
+
+```
+$ make
```
Comment (JoshuaSkelly) [inline README.md]: I don't think we need to mention the make file. Folks who are used to using those will see it and use it.

File: README.md [hunk 7]
```diff
@@ -9,19 +9,26 @@ Turn Twitch chatter into Python events.
 ## Features
 
 - *Pure Python:* No extra dependencies. Just plain and simple Python.
-- *Small API:* With two classes and four methods, you can learn it over a coffee break.
+- *Small API:* With three classes and seven methods, you can learn it over a coffee break.
```
Comment (JoshuaSkelly) [inline README.md]: 👍 Good catch

File: twitchobserver/twitchobserver.py [hunk 8]
```diff
@@ -59,7 +59,7 @@ def __init__(self, nickname, password, channel=None):
         self._is_running = True
         self._socket = None
         self._inbound_polling_rate = 30 / 40
-        self._outbound_send_rate = 30 / 20
+        self._outbound_sending_rate = 30 / 20
```
Comment (JoshuaSkelly) [inline twitchobserver/twitchobserver.py]: Maybe flip this?
- self._inbound_poll_rate()
- self._outbound_send_rate()

File: README.md (hunk 6 above)
Comment (PythooonUser) [inline README.md]: Okay

File: README.md (hunk 5 above)
Comment (PythooonUser) [inline README.md]: I personally like if people provide a way/information on to remove software. Not just how to get it installed. But I get your point. It should be obvious.

**Comment by PythooonUser** [comment]
All those calculations operate on times. To convert a rate to a time you take the invers.
Do I missunderstand something here with the rates? Please help me out.

**Comment by JoshuaSkelly** [comment]
I inverted the calculation in commit: a5b031cf2d406020a07002bb9e9d139ba1216770
The send limit is 20 messages / 30 seconds. Which is 2/3 messages per second, or 3/2 seconds per message.

**Comment by PythooonUser** [comment]
Oh I see. My bad. Okay!

**Comment by JoshuaSkelly** [comment]
No biggie! I made the original change after you pointed it out to me. 😄

**Comment by PythooonUser** [comment]
Done.

**Comment by PythooonUser** [comment]
Just a thought: Since now it is not a rate anymore, maybe call it time_step? But actually it is not that of a big deal, so we should leave it as it is :)

**Comment by JoshuaSkelly** [comment]
You are correct. I think the correct term is interval. What do you think?

**Comment by PythooonUser** [comment]
Yes. That makes me happy :) Made the change.

**Comment by JoshuaSkelly** [comment]
Oops. Only one 'l' interval

**Comment by PythooonUser** [comment]
Shit^^ In German it is two.

**Comment by PythooonUser** [comment]
No, you can merge.

**Pull request merged**